src/components/Testimonials/Testimonials.css

```
@@ -0,0 +1,131 @@
+    .testimonial-card {
+        width: 280px;
+        padding: 1.5rem;
+    }
+}
+
+/* Reduce Motion */
+@media (prefers-reduced-motion: reduce) {
+    .testimonials-wrapper {
+        animation: none;
+        transform: none !important;
+        /* Override GSAP */
```

Stylelint (17.2.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)